polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -44,6 +45,111 @@ using namespace mlir::polygeist;
 
 namespace {
 
+using ValueOrUnsignedTy = std::variant<Value, unsigned>;
+class ValueOrUnsigned {
+public:
+  /// Return a Value or unsigned of \p val depending on \p createVal.
```

Comment:
Given that this class is in a cpp file and the implementation of these member fuctions is simple I think you can remove the comments.

Reply:
removed.